QualityControl/lib/repositories/LayoutRepository.js

```diff
@@ -29,6 +29,34 @@ export class LayoutRepository extends BaseRepository {
             && (filter.name === undefined || layout.name === filter.name));
   }
 
+  /**
+   * Retrieves simplified layout card objects containing only essential properties, optionally filtered
+   * @param {object} filter - Filter criteria with optional keys:
+   *   * owner_id {number} - Filter by owner ID
```

Comment:
To be more clear what is being used from the filter, I wold recommend deconstructing the JSON in the method declaration.
Then the docs could also be updated similar to:
```js
* @param {object} filter - Filter criteria with optional keys:
* @param {string} filter.name - Name of the layout that is to be filtered by
* @param {string} filter.owner_id - ID of the owner of the layout to filter by
[...]
listLayoutCards({name = undefined, owner_id = undefiend) {
```